contract/r/gnoswap/gov/staker/delegate_undelegate_test.gno

```diff
@@ -0,0 +1,110 @@
+package staker
+
+import (
+	"std"
+	"testing"
+
+	"gno.land/p/demo/avl"
+)
+
+func TestDelegateInternal(t *testing.T) {
+	t.Skip("Must running separately. because this test depends on TestUndelegate's state")
```

Comment:
Leftover?

Reply:
skipped because it was affected by the previous test.